nx/source/services/hid.c

```diff
@@ -61,6 +76,29 @@ void hidExit(void)
     shmemClose(&g_hidSharedmem);
 }
 
+void hidReset(void)
+{
+    // Reset internal state
+    memset(&touchEntry, 0, sizeof(HIDTouchScreenEntry));
+    memset(&mouseEntry, 0, sizeof(HIDMouseEntry));
+    memset(&keyboardEntry, 0, sizeof(HIDKeyboardEntry));
+    memset(&controllerEntries, 0, sizeof(HIDControllerInputEntry));
```

Comment:
This is an array not a single entry: remove '&' and use sizeof(controllerEntries).